sdk/resourcemanager/azure-resourcemanager-servicebus/src/main/java/com/azure/resourcemanager/servicebus/implementation/QueuesImpl.java

```diff
@@ -49,7 +51,7 @@ public QueueImpl define(String name) {
     public Mono<Void> deleteByNameAsync(String name) {
         return this.innerModel().deleteAsync(this.resourceGroupName,
                 this.namespaceName,
-                name);
+                name.replaceAll(Pattern.quote("/"), "~"));
```

Comment:
May not do name replacement for customer in SDK.

Reply:
OK. Leave it be. Let user use "~"